### ISSUE TITLE
Forced reinstall OpenHAB after update DSM from 6.1.3-15152 Update 5 to DSM 6.1.3-15152 Update 6

### ISSUE THREAD

**Issue by loovanloon** [openhab/openhab-syno-spk#102]
I had to reinstall openHAB-2.2.0.007-SNAPSHOT after updating DSM from 6.1.3-15152 **Update 5** to DSM 6.1.3-15152 **Update 6** because the installed openHAB package couldn't be started in the packages manage. 

I removed openHAB-2.2.0.007-SNAPSHOT and reinstalled it.

![image](https://user-images.githubusercontent.com/25280600/31433163-c11e7d60-ae78-11e7-8133-75cfab701144.png)

**Comment by cniweb** [comment]
I have no DSM 6.1.x and can not test it.
I am dependent on your help.
Would be good if someone could look at that, thanks!

**Comment by SchoppAx** [comment]
Yeah, there is a bug if the system has to restart the openhab installation.

So for example if you running openhab and restart your system, openhab is still be running (in my case), but you can't manage (start/stop) it anymore.
This occurs because the PID file isn't updated.

We need a better mechanism for that case.

**Comment by loovanloon** [comment]
I can confirm the OH2 package was running when updating the DSM and that update included a reboot.

**Comment by cniweb** [comment]
@kaikreuzer Same problem as here https://github.com/openhab/openhab-docker/issues/108 ???
Pid file Handling?

**Comment by cniweb** [comment]
We are in the process of adjusting PID file handling.
